manager/gui/tests/unit/LrSystemHealthConfiguration.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { mount, createLocalVue } from "@vue/test-utils";
+import LrSystemHealthConfiguration from "@/components/configuration/LrSystemHealthConfiguration.vue";
+import Vuetify from "vuetify";
+import { Vue } from "vue-property-decorator";
+
+describe("LrSystemHealthConfiguration", () => {
+  let wrapper: any;
+  const localVue = createLocalVue();
+  let vuetify: Vuetify;
+
+  beforeAll(() => {
+    vuetify = new Vuetify();
+    wrapper = mount(LrSystemHealthConfiguration, { localVue, vuetify });
+  });
+
+  test(`Title should be "System Health"`, () => {
+    const pageTitle = wrapper.get(".page-title");
+    expect(pageTitle.text()).toMatch("System Health");
+  });
+
+  test(`Presence of protocol dropdown field`, () => {
+    const protocolDropdown = wrapper.find(
+      `[data-test="protocol-dropdown-input"]`
+    );
+    expect(protocolDropdown.exists()).toBe(true);
+  });
+
+  test(`Presence of server input field`, () => {
+    const serverInput = wrapper.find(`[data-test="server-input"]`);
+    expect(serverInput.exists()).toBe(true);
+  });
+
+  test(`Presence of port input field`, () => {
+    const portInput = wrapper.find(`[data-test="port-input"]`);
+    expect(portInput.exists()).toBe(true);
+  });
+
+  test(`Presence of email input field`, () => {
+    const emailInput = wrapper.find(`[data-test="email-input"]`);
+    expect(emailInput.exists()).toBe(true);
+  });
+
+  test(`Presence of password input field`, () => {
+    const passwordInput = wrapper.find(`[data-test="password-input"]`);
+    expect(passwordInput.exists()).toBe(true);
+  });
+
+  test(`Presence of confirm password input field`, () => {
+    const confirmPasswordInput = wrapper.find(
+      `[data-test="confirm-password-input"]`
+    );
+    expect(confirmPasswordInput.exists()).toBe(true);
+  });
+
+  test(`Presence of receiver emails input field`, () => {
+    const receiverEmailsInput = wrapper.find(
+      `[data-test="receiver-emails-input"]`
+    );
+    expect(receiverEmailsInput.exists()).toBe(true);
+  });
+
+  test(`Presence of 'Test' button.`, () => {
+    const testButton = wrapper.find(`[data-test="test-btn"]`);
+    expect(testButton.exists()).toBe(true);
+  });
+
+  test(`Presence of verified checkbox field`, () => {
+    const verifiedCheckbox = wrapper.find(`[data-test="verified-checkbox"]`);
+    expect(verifiedCheckbox.exists()).toBe(true);
+  });
+
+  test(`Presence of 'Apply' button`, () => {
+    const applyButton = wrapper.find(`[data-test="apply-btn"]`);
+    expect(applyButton.exists()).toBe(true);
+  });
+
+  test(`Presence of 'Reset' button`, () => {
+    const resetButton = wrapper.find(`[data-test="reset-btn"]`);
+    expect(resetButton.exists()).toBe(true);
+  });
+
+  test(`Validation of server input field`, async () => {
+    const serverInput = wrapper.get(`[data-test="server-input"]`);
+    await serverInput.setValue("test value");
+    await Vue.nextTick();
+
+    expect(wrapper.text()).toContain("Invalid value");
```

Comment:
This only tests one validation, you can create test case for validating all the scenario for that field or do we need separate test case is debatable.